lib/widgets/screens/profile/profile.dart

```diff
@@ -177,36 +186,39 @@ class _ProfileScreenScrollState extends State<ProfileScreenScroll> {
               },
             ),
           ProfileBasicInfo(
-            userType: widget.userData.offersHelp
-                ? UserType.mentor
-                : UserType.entrepreneur,
-            fullName: widget.userData.fullName!,
-            avatarUrl: widget.userData.avatarUrl,
-            pronouns: "they/them", //TODO
+            userType:
+                userData.offersHelp ? UserType.mentor : UserType.entrepreneur,
+            fullName: userData.fullName!,
+            avatarUrl: userData.avatarUrl,
+            pronouns: userData.pronounsDisplay.isNotEmpty
+                ? userData.pronounsDisplay
+                : null,
             affiliations: const ["Verizon Digital Ready"], //TODO
             company: company.name,
-            companyRole: widget.userData.jobTitle,
-            education: widget.userData.educationLevel?.translatedValue,
-            linkedinUrl: "https://www.linkedin.com/in/williamhgates/", //TODO
+            companyRole: userData.jobTitle,
+            education: userData.educationLevel?.translatedValue,
```

Comment:
Is this used at all? is it for titles (e.g. "Dr. First Last")?

Reply:
@andersan 

We are displaying this field in the profile underneath the job title. I am not sure if we should use the latest business experience and academic experience for these fields, or if the user will be able to pick the job and school to show here.

It is still not super clear to me how the User.jobTitle differs from the BusinessExperience.jobTitle, and which one to use in the quick view card.

Reply:
We may combine companies and `BusinessExperience` later on and just have one object type, but the requirements are a bit unclear. If we do combine those, we can remove `User.jobTitle`.

Hopefully after the onboarding flow is defined it'll be clearer how this data should be organized.